### ISSUE TITLE
A question about Jupyter, Twitter, and our social media strategy

### ISSUE THREAD

**Issue by afshin** [jupyter-governance/ec-team-compass#9]
We are working on our communication strategy on social media, and in particular whether we should use Twitter. If you would like to share your thoughts with the Jupyter Executive Council, please consider one of these three channels:

* Responding privately to this [three-question poll](https://forms.gle/GHruQEYj9b9TPmyX6).
* Commenting publicly on this issue.
* Joining us at the next Executive Council office hours call (on 13 July at 10am US Pacific on [`jovyan` Zoom channel](https://www.google.com/url?q=https://zoom.us/j/2264645576?pwd%3Dc0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09&sa=D&source=calendar&usd=2&usg=AOvVaw2DFXjugRh3bV_LDUpnhjd1)).

**Comment by mwcraig** [comment]
Having a social media presence somewhere can be useful for announcements/advertising longer form things like blog posts. I have no idea what the right one is these days. 

Ceasing to advertise the twitter channel makes sense. 

Eventually posting a last tweet saying where to find the project in the future also makes sense.

As someone pointed out in the original issue, I think it would be a mistake to delete the account because someone else could grab the name then.

**Comment by astrojuanlu** [comment]
Any decision regarding Twitter has to take into account that inactive profiles may be deleted very quickly (30 days without login) https://help.twitter.com/en/rules-and-policies/inactive-twitter-accounts

Having said that, I think people need to come to terms with the fact that Twitter has no future under current leadership. And I'm not optimistic that leadership will be replaced, given that it's a private company now.

There are several well maintained tech instances running Mastodon but also Calckey and very soon Takahe as well (Django based https://github.com/jointakahe/takahe) surely there are UX issues, but comparatively Bluesky is much worse (this can of course change soon, but so can Mastodon and other Fediverse apps).

Jupyter has always advocated the democratization of open science, and choosing a social media strategy that includes community-owned infrastructure seems like the right path. The Fediverse is not "mainstream" yet and it's unclear if it will ever be in the sense that Twitter or Instagram are, but it has reached critical mass.

Lots of us are waiting for you folks. Hopefully see you soon on the Fediverse.

**Comment by ashwinvis** [comment]
I would encourage Jupyter to open an account in a Mastodon / Fediverse instance.

Several Pythonistas and CPython core developers, your core audience (scientists, researchers, programmers and data scientists) are already present there.

**Comment by stark1tty** [comment]
Twitter is becoming obsolete, only federated social media offers some stability or longevity.

**Comment by ivanov** [comment]
I'm fine if our main account on Twitter is de-emphasized, not advertised, or not listed at all on jupyter.org website.

It's unfortunate that twitter has transformed into a walled garden, where no one can read anything without being logged in, but  Twitter, as a medium, still has quite a large following within our community. For example the #scipy2023 tweets from this week, while many have impression counts in the low hunderds, there are also a few dozen that have thousands of views. These numbers are still quite big given how many people have left twitter, and can no longer read anonymously.

**Comment by ivanov** [comment]
@ashwinvis wrote:
> I would encourage Jupyter to open an account in a Mastodon / Fediverse instance.

We've had some suggestion and discussion on this issue: https://github.com/jupyter/governance/issues/146

**Comment by astrojuanlu** [comment]
Was this discussed at the Executive Council office hours call on 13 July?

**Comment by blink1073** [comment]
Hi @astrojuanlu!  Yes, the meeting notes are [public](https://docs.google.com/document/d/1N5jm0VJlDUX4WK0n7Ia8Ff63RdzNYkH28PkSZPdBtUc/edit#heading=h.ndx1n9hc8yl9).  

tl;dr We should generally use social media as a broadcast channel.  Original content should be on our blog.  Jupyter Discourse is for general discussion.  We may decide to adopt a Mastodon account and engage more there once we finish standing up the Jupyter Media Strategy Working Group.
